app/models/requirement.rb

```diff
@@ -92,5 +93,9 @@ def valid_title_or_skill
         errors.add(:title, "must have a title or a skill")
         errors.add(:skill, "must have a title or a skill")
       end
+      if !title.blank? && !skill.blank?
```

Comment:
Would `if title.present && skill.present?` be better, get rid of the negatives ?